docs/sources/configure/examples/thanos-storage-configs.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Configuration examples for using Thanos-based storage clients"
+description: "Minimal examples for using Thanos-based S3, GCS, Azure, and filesystem clients in Grafana Loki."
+weight: 100
+---
+
+# Configuration examples for using Thanos-based storage clients
+
+Use these examples as a starting point for configuring [Thanos based object storage clients](https://grafana.com/docs/loki/<LOKI_VERSION>/configure/#thanos_object_store_config) in Grafana Loki.
+
+## GCS example
+```yaml
+storage_config:
+  use_thanos_objstore: true
+  object_store:
+    gcs:
+      bucket_name: my-gcs-bucket
+
+      # JSON either from a Google Developers Console client_credentials.json file,
```

Comment:
This comment is confusing to me.  Does this whole comment apply to the `service_account`?  Shouldn't the input be a `file.json` of some sort?  
Is there a typo at line 30?  Should it read `service_account: service-account.json` instead?

Reply:
> JSON either from a Google Developers Console client_credentials.json file,
> or a Google Developers service account key. Needs to be valid JSON, not a
> filesystem path.

only this portion of the comment applies to the `service_account` field. It expects the json contents, not a file path.

Rest of the comment mentions how the google SDK picks up credentials either from env or known file locations.

Reply:
i took this content from our configuration reference, didn't trim it down for the sake of completeness. we could rewrite this in both places if its confusing :)
 https://grafana.com/docs/loki/next/configure/#thanos_object_store_config

Reply:
I think I read this over half a dozen times, and while the commented text made sense to me, I was having trouble match it to the parameter.  But reading it over again, I think the commented text is clear enough, i was just stuck trying to match it to the example parameter.

If you don't want to rewrite the comment, maybe change line 31 to read
`service_account: your-service-account.json` ?